src/scripts/configs/routes.js

```diff
@@ -1,5 +1,6 @@
 import * as SurahsActions from 'actions/SurahsActions';
 import * as AyahsActions from 'actions/AyahsActions';
+import RouteNotFound from 'utils/RouteNotFound';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>'RouteNotFound' is defined but never used.